docs/UserGuide.md

```diff
@@ -139,8 +162,21 @@ Adds a person to AronaPro.
 Format: `add id/NUSID n/NAME p/PHONE_NUMBER e/EMAIL t/TAG [g/GROUP]…​`
 
 > Note:
+> * The `NUSID` refers to the NUSID shown in the displayed person list.
 > * The `NUSID` **must be a 7-digit number following an 'E'**.
+> * `NAME` can consist of numbers.
+> * `NAME` can be duplicated for different persons.
 > * `PHONE_NUMBER` has to be 3-10 digits long.
+> * `EMAIL` can be duplicated for different persons.
+> * `EMAIL` follows the following format and constraints:
+  >   * It should have a local-part followed by a `@` and then a domain name.
+>     * Local-part should only contain alphanumeric characters and special characters(+ _ . -).
+>     * Local-part may not start or end with any special characters.
+>     * Domain name is made up  of domain label separated by periods.
+  >     * Domain name must end with a domain label at least 2 characters long.
+>       * Have each domain label start and end with alphanumeric characters.
+>       * Have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
+> * For `TAG``, the valid forms have to be either 1 of these:  Professor, TA, Student, None.
```

Comment:
Might need to specify (Case-sensitive)
If not already done elsewhere. 
(Edit: already specified below lol, but ig if you want you can sneakily save one known issue by clearly stating here that it's case sensitive)